index.html

```diff
@@ -0,0 +1,143 @@
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Capstone Project Module 1: 40 Startup Ideas to Try in 2022</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://use.typekit.net/dol5hyu.css">
+  <script src="script.js" defer></script>
+</head>
+
+<body>
+
+  <!--------------------- The header section------------ -->
+  <article class="header-section">
+    <header class="main-page">
+      <nav class="mobile-hidden social-media">
+        <ul>
+          <li><a href="https://www.facebook.com/kashif.saleem.9"><img class="fb-image" src="images/facebook-f.svg"
+                alt="facebook-icon"></a></li>
+          <li><a href="https://twitter.com/Kashif14Saleem"><img class="twitter-image" src="images/twitter.svg"
+                alt="facebook-icon"></a></li>
+          <li><button class="button-nav" type="button"> English</button></li>
+          <li><button class="button-nav" type="button"> Mypage</button></li>
+          <li><button class="button-nav" type="button"> Logout</button></li>
+        </ul>
+      </nav>
+      <ul class="desktop-navigation">
+        <li><img class="startp-logo-nav mobile-hidden"
+            src="images/startup-business-conference-high-resolution-logo-color-on-transparent-background.png"
+            alt="company-logo"></li>
+        <li class="margin-top nav-item"><a href="about.html" class="nav-link">About</a></li>
+        <li class="nav-item"><a href="#desktop-portfolio" class="nav-link">Program</a></li>
+        <li class="nav-item"><a href="#desktop-about" class="nav-link">Join</a></li>
+        <li class="nav-item"><a href="#desktop-contact" class="nav-link">Sponsor</a></li>
+        <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+        <li class="nav-item desktop-nav-item"><a href="#" class="nav-link">SBC Campaign</a></li>
+      </ul>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </header>
+    <section>
+      <div class="hero-container">
+        <h1 class="Hero-H1 ">"Hello! Aspiring enterpreneues"</h1>
+        <span class="h1-paragraph">40 Startup Business Ideas
+          Conference 2022</span>
+      </div>
+      <div class="para-containerd">
+        <p class="hero-paragraph">
+          Join us in LAHORE for the "Startups 2022" confress to discover countless oppurtunities and benefits of
+          self-employment!
+        </p>
+      </div>
+      <h2 class="Date-container">
+        <span class="date">2022.11.25(FRI) ~ 27(SUN)</span>
+        <span class="location">@ Alhamra arts council, 68- Mall Road, Lahore</span>
+      </h2>
+    </section>
+  </article>
+  <!--------------------- The header section------------ -->
+
+  <!--------------------- The herosection------------ -->
+  <main>
+    <section class="Main-program">
```

Comment:
- **Hello, I think you did not understand what the Previous reviewer requested. Have a look at these pictures below:** 
- Original Design
- 
![Capture1](https://user-images.githubusercontent.com/63972670/203847061-40b5ad66-67f3-4be1-96d5-f101470b8c0d.PNG)
- Look at your Design: 
- 
![Capture](https://user-images.githubusercontent.com/63972670/203847125-99ecc70e-14b6-4577-afed-1575db098060.PNG)
- **Please Open This link and follow 👉  the[ design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), make your work to be like what in the original.**

Reply:
@OLIPLICHE  this is what the previous reviewer asked me for " **please make sure that the boxes should be transparent as the ones in the Design in order to match the design**"

So that is what I exactly did. I made the background transparent.
Moreover, you made some lines and arrows with question marks...Can you please be specific about what you are trying to suggest? 
Although the changes you are requesting are not in line with the previous reviewer's comments, I will still try to make the changes and submit them again.

Reply:
- Sorry, for the late reply. I see that your project has been approved. Congratulations. The main goal of this project is that your website should look like the original design. That is why I sent you the pictures and the Link where you can find the original design and Apply the change. Sorry for any confusion, I wish you the best. @Kashif-Saleem-Ghuman